Makefile

```diff
@@ -319,9 +319,9 @@ endif
 build-windows:
 	@echo "===> Building Antrea bins and antrea/antrea-windows Docker image <==="
 ifneq ($(NO_PULL),)
-	docker build -t antrea/antrea-windows:$(DOCKER_IMG_VERSION) -f build/images/Dockerfile.build.windows $(DOCKER_BUILD_ARGS) .
+	docker build -t antrea/antrea-windows:$(DOCKER_IMG_VERSION) -f build/images/Dockerfile.build.windows.test $(DOCKER_BUILD_ARGS) .
```

Comment:
Why do you want to use a name with suffix "test" in your final PR?

Reply:
Draft pr has been updated.